.travis.yml

```diff
@@ -1,124 +0,0 @@
-# Travis caches can be manually removed if necessary. This is one of the very
```

Comment:
We may want to keep Travis active temporarily to compare the behavior. After the project gets some experience with GitHub Actions, we can turn it off permanently.

Reply:
Good idea, fixed.